### ISSUE TITLE
Error: Unable to parse color from string: noColor

### ISSUE THREAD

**Issue by MarcelBolten** [TeselaGen/openVectorEditor#747]
Hi @tnrich,

Using a [.dna file](https://media.addgene.org/snapgene-media/v1.7.9-0-g88a3305/sequences/306280/b6631434-e8c7-40c7-b4d2-903a1b3d5c48/addgene-plasmid-155130-sequence-306280.dna) from https://www.addgene.org/155130/sequences/ I am given an "Unable to parse color from string: noColor" error in the Firefox console when switching from circular map to linear map and scrolling around:
```
Error: Unable to parse color from string: noColor
    l https://demo.elabftw.net/assets/ove/open-vector-editor.min.js?v=4.0.7:55
    l https://demo.elabftw.net/assets/ove/open-vector-editor.min.js?v=4.0.7:55
    render https://demo.elabftw.net/assets/ove/open-vector-editor.min.js?v=4.0.7:41
```
(see [pastebin for more details](https://pastebin.com/YkKDXPpm))
Error is given also on the demo server with the same file.
```
Error: Unable to parse color from string: noColor
    d http://teselagen.github.io/openVectorEditor/demo.6706732a.js:55
    d http://teselagen.github.io/openVectorEditor/demo.6706732a.js:55
    render http://teselagen.github.io/openVectorEditor/demo.6706732a.js:221
```

**Comment by tnrich** [comment]
Hey @MarcelBolten thanks for the error report. I've fixed this and published a new version open-vector-editor@16.3.3. Lemme know if that works for you. 

-Thoams

**Comment by MarcelBolten** [comment]
Hi @tnrich thanks for the fix. Works flawless now.
